### PR TITLE
Core200s

### PR DESCRIPTION
**Pull request by gofaster**
Adding initial support for the Levoit Core 200S air purifier.  Methods replicates most of the local controls except the Timer button.
In relation to https://github.com/webdjoe/pyvesync/issues/68

### CODE REVIEW

File: src/pyvesync/vesyncfan.py [hunk 1]
```diff
@@ -11,6 +11,381 @@
...
+        self.enabled = True
+        self.details: Dict[str, Union[str, int, float]] = {
+            'filter_life': 0,
+            'mode': "manual",
+            'level': 0,
+            'display': False,
+            'child_lock': False,
+            'night_light': "off"
+        }
+        self.config: Dict[str, Union[str, int, float]] = {
```
Comment (webdjoe) [inline src/pyvesync/vesyncfan.py]: Should read `Union[str, int, float, bool]`

File: src/pyvesync/vesyncfan.py [hunk 2]
```diff
@@ -11,6 +11,381 @@
...
+        r, _ = Helpers.call_api(
+            '/cloud/v2/deviceManaged/bypassV2',
+            method='post',
+            headers=head,
+            json=body,
+        )
+        outer_result = r.get('result', {})
+        inner_result = None
+
+        if outer_result is not None:
```
Comment (webdjoe) [inline src/pyvesync/vesyncfan.py]: `if outer_result:`

File: README.md [hunk 3]
```diff
@@ -20,13 +20,15 @@ pyvesync is a library to manage VeSync compatible [smart home devices](#supporte
...
   - [Levoit 300S humidifer](#levoit-humidifier-300s-methods-and-properties)
+  - [Levoit Core200S air purifier Methods and Properties](#levoit-purifier-core200s-methods-and-properties)
   - [JSON Output API](#json-output-api)
     - [JSON Output for All Devices](#json-output-for-all-devices)
     - [JSON Output for Outlets](#json-output-for-outlets)
     - [JSON Output for Dimmable Switch](#json-output-for-dimmable-switch)
     - [JSON Output for Bulbs](#json-output-for-bulbs)
     - [JSON Output for Air Purifier](#json-output-for-air-purifier)
     - [JSON Output for 300S Humidifier](#json-output-for-300s-humidifier)
+	- [JSON Output for Core200S Purifier](#json-output-for-core200s-purifier)
```
Comment (webdjoe) [inline README.md]: One less indent

**Review by webdjoe — COMMENTED** [review]
This is amazing, thank you for the PR! Just a few minor fixes and I will merge.

**Comment by gofaster** [comment]
Thank you for reviewing and catching the errors.  Made the changes and retested against my Core 200S.

**Pull request merged**